### DISCUSSION TITLE
How to return only one attribute of an object

### DISCUSSION THREAD

**Discussion by J-Christophe** [tortoise/tortoise-orm#810] [Q&A]
Hi,

I am beginner with Tortoise/Pydantic and I spent a lot of time to find how to return only one attribute of an object but without success. Based on the example that is provided in the documentation, I would like to return only the name of the user and not the full object. Could you explain me how to do please ?

@app.get(
    "/user/{user_id}", response_model=User_Pydantic, responses={404: {"model": HTTPNotFoundError}}
)
async def get_user(user_id: int):
    return await User_Pydantic.from_queryset_single(Users.get(id=user_id))

Thanks for your support
Jean-Christophe

**Comment by long2ice** [comment]
use `queryset.values()`
